prow/jobs/incubator/compass/components/runtime-agent/runtime-agent.yaml

```diff
@@ -0,0 +1,80 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  skip_report: false
+  optional: true
+  decorate: true
+  path_alias: github.com/kyma-incubator/compass
+  max_concurrency: 10
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/buildpack-golang:go1.11
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-incubator/compass/components/runtime-agent"
+        resources:
+          requests:
+            memory: 1.5Gi
+            cpu: 0.8
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-incubator: "true"
+
+presubmits: # runs on PRs
+  kyma-incubator/compass:
+    - name: pre-master-compass-components-runtime-agent
+      branches:
+        - master
+      <<: *job_template
+      run_if_changed: "^components/runtime-agent/"
+      extra_refs:
+        - <<: *test_infra_ref
+          base_ref: master
+      labels:
+        <<: *job_labels_template
+        preset-build-pr: "true"
+    - name: pre-rel12-compass-components-runtime-agent
```

Comment:
This component did not exist on release 1.2 and 1.3, therefore, the release jobs should be deleted.